test/test-server/kafka_rest_router.go

```diff
@@ -109,6 +109,9 @@ func handleKafkaRestClusters(t *testing.T) http.HandlerFunc {
 // Handler for: "/kafka/v3/clusters/{cluster}/acls"
 func handleKafkaRestACLs(t *testing.T) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		clusterName := vars["cluster"]
```

Comment:
Looks like this is a cluster ID if it's sometimes equal to "lkc-acls"
```suggestion
		clusterId := vars["cluster"]
```